client/my-sites/domains/domain-management/edit/card/header/primary-domain-button.jsx

```diff
@@ -38,18 +31,35 @@ const PrimaryDomainButton = createReactClass( {
 			label = this.props.translate( 'Make Primary' );
 
 			return (
-				<Button
-					compact
-					className="domain-details-card__make-primary-button"
-					onClick={ this.handleClick }
-				>
+				<Button compact onClick={ this.handleClick }>
 					{ label }
 				</Button>
 			);
 		}
 
 		return null;
-	},
-} );
+	}
+}
+
+const makePrimaryClick = domain =>
```

Comment:
We could rename that function in to `recordMakePrimaryClick()` to better convey its purpose.

Reply:
done